assets/flutter_i18n/en.json

```diff
@@ -8,6 +8,17 @@
   "home_expositionprogression_since": "From",
   "home_places_title": "Locations I've visited today",
 
+  "home_info_operation_title": "How does it work?",
+  "home_info_operation_text1": "Pandemia offers several features assisting you in your daily movements during a pandemic.",
+  "home_info_operation_text2": "It computes your daily exposition and broadcast rates, and allows you to consult popularity of your favorite places, to find out when it's the safest time to go there.",
+  "home_info_data_title": "Personal data",
+  "home_info_data_text1": "Pandemia is a standalone application; it does not send your data to third parties",
+  "home_info_data_text2": "If you activate your smartphone's geolocation, we use locations to improve our calculations' results; these locations are stored on our smartphone, and nowhere else.",
+  "home_info_disclaimer_title": "Disclaimer",
+  "home_info_disclaimer_text": "Pandemia-computed information does not replace screening tests, but can help you decide to be tested with health professionals.",
```

Comment:
"whether to be tested by health services" instead of "to be tested with health professionals."